Jellyfin.Server/Migrations/Routines/MigrateRatingLevels.cs

```diff
@@ -41,54 +39,33 @@ public MigrateRatingLevels(
         /// <inheritdoc/>
         public void Perform()
         {
-            var dbPath = Path.Combine(_applicationPaths.DataPath, DbFilename);
-
-            // Back up the database before modifying any entries
-            for (int i = 1; ; i++)
+            _logger.LogInformation("Recalculating parental rating levels based on rating string.");
+            using var context = _provider.CreateDbContext();
+            using var transaction = context.Database.BeginTransaction();
+            var ratings = context.BaseItems.Select(e => e.OfficialRating).Distinct();
+            foreach (var rating in ratings)
             {
-                var bakPath = string.Format(CultureInfo.InvariantCulture, "{0}.bak{1}", dbPath, i);
-                if (!File.Exists(bakPath))
+                if (string.IsNullOrEmpty(rating))
                 {
-                    try
-                    {
-                        File.Copy(dbPath, bakPath);
-                        _logger.LogInformation("Library database backed up to {BackupPath}", bakPath);
-                        break;
-                    }
-                    catch (Exception ex)
-                    {
-                        _logger.LogError(ex, "Cannot make a backup of {Library} at path {BackupPath}", DbFilename, bakPath);
-                        throw;
-                    }
+                    context.Database.ExecuteSqlRaw("UPDATE BaseItems SET InheritedParentalRatingValue = NULL WHERE OfficialRating IS NULL OR OfficialRating='';");
```

Comment:
```suggestion
                    context.BaseItems.Where(e => e.OfficialRating == null || OfficialRating == ";").ExecuteUpdate(f => f.SetProperty(e => e.InheritedParentalRatingValue = null));
```
Not tested code but should resemble it close enough.